src/utils/index.ts

```diff
@@ -332,9 +332,10 @@ export const filterMessageListParams = (params: MessageListParams, message: User
   if (params?.customTypes?.length > 0 && !params.customTypes.includes(message.customType)) {
     return false;
   }
-  if (params?.senderUserIds?.length > 0) {
+  if (params?.senderUserIds && params?.senderUserIds?.length > 0) {
```

Comment:
Lets add unit test

Reply:
I think filterMessageListParams function is tested on the other tests for reducers